### DISCUSSION TITLE
Plot file with different farmer key

### DISCUSSION THREAD

**Discussion by Wedaras** [Chia-Network/chia-blockchain#5222] [Support]
Hi, i have plotted one plot with another PC and transferred to final directory on my NAS. The node sees it but wenn i run check plots on cmd, it says that the new plot "has a farmer public key that is not in the farmer's pk list" is there any possibility to add the public key to the farmer's pk list? Thank you in advance for your help.
Best wisches farmer Andrew :)

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
